pupil_src/shared_modules/file_methods.py

```diff
@@ -45,7 +45,7 @@ def close(self):
 def _load_object_legacy(file_path):
     file_path = os.path.expanduser(file_path)
     with open(file_path, 'rb') as fh:
-        data = pickle.load(fh, encoding='bytes')
+        data = pickle.load(fh, raw=True)
```

Comment:
Is this change intentional?

Reply:
new zmq api.

Reply:
recommended change.

Reply:
nvermind. you are right. This was a mistake.